index.js

```diff
@@ -7,8 +7,36 @@ class SQLStatement {
    * @param {any[]} values
    */
   constructor(strings, values) {
-    this.strings = strings
-    this.values = values
+    if (values.length === 0) {
+      this.strings = strings
+      this.values = values
+    } else {
+      this.strings = []
+      this.values = []
+      let prevOpened = false
```

Comment:
It means that previous value of `this.strings` array was created by `append` function, and next value of `strings` should be appended to previous value of `this.strings`, instead of creating new one